### PR TITLE
gui/lang: Updated translations and fixed typos.

### PR DESCRIPTION
**Pull request by Croden1999**
- gui/lang: Updated translations and fixed typos.
- data/fonts: Add a font file, and also included license. 

Please devs review this PR.

### CODE REVIEW

File: vita3k/gui/src/gui.cpp [hunk 1]
```diff
@@ -213,8 +213,8 @@ static void init_font(GuiState &gui, EmuEnvState &emuenv) {
     ImFontGlyphRangesBuilder builder;
     builder.AddRanges(io.Fonts->GetGlyphRangesJapanese());
     builder.AddRanges(extra_range);
-    ImVector<ImWchar> japanes_and_extra_ranges;
-    builder.BuildRanges(&japanes_and_extra_ranges);
+    ImVector<ImWchar> japanese_and_extra_ranges;
```
Comment (Zangetsu38) [inline vita3k/gui/src/gui.cpp:216]: Oupsy 🤦‍♂️

**Comment by nishinji** [comment]
Is it too much of a disadvantage to increase the size of the font file by a factor of ten or more just for Chinese?
Besides, it is only used when no official font exists.
up: master 
down: pr
image

**Comment by Croden1999** [comment]
> Is it too much of a disadvantage to increase the size of the font file by a factor of ten or more just for Chinese? Besides, it is only used when no official font exists. up: master down: pr image

the Noto Sans CJK font include Korean characters, but i no test it.

**Comment by nishinji** [comment]
The problem is not that Chinese is the only language that benefits, but that the build is too large
It would be better to consider using the system's fonts

**Comment by CoffeeBrewer64** [comment]
could this be put into an option (a tick box) in the gui? on setup you could ask if you want to download a font with more languages supported, and when the user switches to chinese and korean, ask if they want to download it as well. it could also be kept as a button in the system config. this way, the build size stays the same and the user still gets the font they want/need.

**Comment by Croden1999** [comment]
> could this be put into an option (a tick box) in the gui? on setup you could ask if you want to download a font with more languages supported, and when the user switches to chinese and korean, ask if they want to download it as well. it could also be kept as a button in the system config. this way, the build size stays the same and the user still gets the font they want/need.

no sense, just for full character range

**Comment by EXtremeExploit** [comment]
I kinda agree with nishinji in this case, it adds a whole 10MB of disk space to the build, its nice to fix the full chinese and korean characters, but is there another font option that doesnt change the build size by about 10MB?

**Comment by Croden1999** [comment]
> I kinda agree with nishinji in this case, it adds a whole 10MB of disk space to the build, its nice to fix the full chinese and korean characters, but is there another font option that doesnt change the build size by about 10MB?

If wanna to be compatible with CJK fonts, only need change the file

**Comment by Croden1999** [comment]
Ready for review

**Comment by Croden1999** [comment]
> I've fixed typos in Indonesian and errors in Lang

if you wanna a translators list, no need add your name in lang.xml file

**Comment by Kitakatarashima** [comment]
Oh yes yes i I'll fix it right away

**Comment by Kitakatarashima** [comment]
Where have you been??

**Comment by Croden1999** [comment]
> Where have you been??

i doing other things. by the way, you can fork the Vita3K repo, create a new branch and switch, upload your lang.xml file(s) and push a new PR, it's very easily.

**Comment by Kitakatarashima** [comment]
It doesn't support YML file

**Comment by Kitakatarashima** [comment]
Sorry, I'm still new to GitHub so I don't really know how to do it

**Review by Zangetsu38 — APPROVED** [review]

**Review by Macdu — APPROVED** [review]
The 10% Vita3K file increase is a bit annoying but I guess it's still reasonable

**Pull request merged**